edc-extensions/business-partner-validation/src/main/java/net/catenax/edc/validation/businesspartner/BusinessPartnerValidationExtension.java

```diff
@@ -22,10 +22,10 @@
 import org.eclipse.dataspaceconnector.policy.model.Duty;
```

Comment:
@carslen could we address this to another PR? This is about release 0.1.5 and I'd like not to add a lot of other effort on it as we're on a rush to getting it delivered.

Reply:
Please adjust at least src files in this PR.

Reply:
@carslen this PR doesn't bring in any new Java file, the ones that are modified are already in the current `main` branch, so I guess none on the file in the `main` branch has the current header, right? Because if this is correct, maybe it's better to fix all the headers in a separate PR to reach a clean state of the codebase in one fell swoop.

Reply:
OK, let's handle license header fix in another PR. To track this, I've opend issue #88